docs/book/src/design/overview.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+The Nautilus Wallet system consists of the following high systems:
+
+1. An underlying [ledger platform](ledger-platform.md) and its asset(s)
+2. The [Wallet](wallet.md) itself:
+   - The [Wallet TEE](wallet-tee.md) (server) hosts and provides access to the Wallet SGX enclave
+   - The [Wallet Connector app](wallet-connector-app.md) (client) securely connects wallet owners with their wallets
+3. [Asset services](asset-services.md), which help manage and govern the asset:
+   - The [KYC service](kyc-service.md)
+   - User verification service
+
+## System Landscape diagram
+
+[![System Landscape diagram][diagram]][diagram]
+
+[diagram]: diagrams/SystemLandscape.svg
```

Comment:
See [my comment](https://github.com/ntls-io/nautilus-wallet/pull/157#issuecomment-1049923365) regarding the imported images in general

Reply:
> The included images seem too detailed and I found them hard to read. I suspect they may better suited for architectural purposes than for developer documentation. What are your thoughts on this?

Well, these docs should be covering the system's architecture, so that's probably in scope, at least?

Regarding the level of detail, we may want to have both simplified and more detailed versions. Structurizr is able to toggle descriptions in its detailed view: I'll see if I can wrangle something with that. (We'll probably want to preserve the more detailed versions somewhere: Co has indicated that that's useful.)

Reply:
I agree, there is value in the more detailed version.  I only think that all that text is probably a bit excessive for the developer docs.  For this specific use case it can probably be trimmed down a bit, like you mention.  Otherwise, is there a way for it to be togglable?

Reply:
The dynamic toggling capability won't be trivial to replicate outside of Structurizr UI, I think.

Reply:
I see, ok, no toggling then.  May I suggest then that we display the simplified version but have a link in the caption (or elsewhere) to the detailed versions?  Unless you know of a better solution that is.